ptbcontrib/roles/roles.py

```diff
@@ -396,7 +396,7 @@ def filter(self, update: Update) -> bool:
         return self.role.filter(update, inverted=True)
 
     def __repr__(self) -> str:
-        return "<inverted {}>".format(self.role)
+        return f"<inverted {self.role}>"
```

Comment:
I dont like that we are doing non related changes in PRs, but its contrib. So I dont care that much. Still wanted to point it out.

Reply:
This is because I introduced `pyupgrade` to the `pre-commit` hooks. I changed this PRs title to "Improve tests & code quality" …

Reply:
ahaha, good fix